server/config/config.go

```diff
@@ -323,6 +333,9 @@ func DefaultConfig() *Config {
 				Keys: []string{"*"},
 			},
 		},
+		Mempool: MempoolConfig{
+			MaxTxs: 0,
```

Comment:
we should bound this by default. Defaults offered by us should always be a bit more conservative than what a node operator may decide to run. 

```suggestion
			MaxTxs: 10_000,
```